src/o365/spo/commands/app/app-retract.ts

```diff
@@ -168,20 +189,25 @@ class SpoAppRetractCommand extends SpoAppBaseCommand {
 
   Remarks:
   
-    To retract an app from the tenant app catalog, you have to first log in to a SharePoint site
+    To retract an app from the tenant or site colleciton app catalog, you have to first log in to a SharePoint site
     using the ${chalk.blue(commands.LOGIN)} command, eg. ${chalk.grey(`${config.delimiter} ${commands.LOGIN} https://contoso.sharepoint.com`)}.
 
-    If you don't specify the URL of the tenant app catalog site using the ${chalk.grey('appCatalogUrl')} option,
-    the CLI will try to determine its URL automatically. This will be done using SharePoint Search.
-    If the tenant app catalog site hasn't been crawled yet, the CLI will not find it and will
-    prompt you to provide the URL yourself.
+    When adding an app to the tenant app catalog, it's not necessary to specify
+    the tenant app catalog URL. When the URL is not specified, the CLI will
+    try to resolve the URL itself. Specifying the app catalog URL is required
+    when you want to add the app to a site collection app catalog.
+
+    When specifying site collection app catalog, you can specify the URL either
+    with our without the ${chalk.grey('AppCatalog')} part, for example
+    ${chalk.grey('https://contoso.sharepoint.com/sites/team-a/AppCatalog')} or
+    ${chalk.grey('https://contoso.sharepoint.com/sites/team-a')}. CLI will accept both formats.
 
-    If the app with the specified ID doesn't exist in the tenant app catalog, the command will fail
+    If the app with the specified ID doesn't exist in the app catalog, the command will fail
     with an error.
    
   Examples:
   
-    Retract the specified app from the tenant app catalog. Try to resolve the URL
+    Retract the specified app from the tenant or site app catalog. Try to resolve the URL
```

Comment:
This is actually incorrect, isn't it? If no scope is specified, the command will try to retract the app from the tenant app catalog, but not both.